src/remembrall/classes/Fields.py

```diff
@@ -72,3 +72,16 @@ def __init__(self, value: str):
 
     def __str__(self):
         return datetime.strftime(self.value, format)
+    
+class Address(Field):
+    def __init__(self, value: str):
+        self.value = value.strip()
```

Comment:
тут повинна бути валідація, бо при створенні об'єкта, сам об'кт повинен себе перевіряти, а не ми пам'ятати
```suggestion
       Address.check_address(value.strip())
        self.value = value.strip()
```

Reply:
Добре, дякую, буде зроблено.